02_getting_started.md

```diff
@@ -10,7 +10,7 @@ headings:
     image: /assets/images/icons/new-user.svg
     excerpt: Step by step introduction to SeqAn
   - url: "https://docs.seqan.de/seqan3.html"
-    title: Already using SeqAn?
```

Comment:
I kind of prefer this title, but that is just my opinion. :)

Reply:
I agree with @MitraDarja . I don't see that `Reference` is a clearer term for the API documentation. Or am I missing a "that's just how it is called normally?"

And it goes well with the former `New to SeqAn?`

Reply:
I thought so, too.

Reply:
I wouldn't expect a Reference behind "Already using SeqAn?".(And I wasn't able to find it when I was looking for it).

To me a reference is also interesting when I am not already using the piece software.
This is actually the main reason I decided to touch the website xD

But if everyone likes the current title more, I can remove it from the PR.